signac/contrib/project.py

```diff
@@ -264,7 +264,7 @@ def min_len_unique_id(self):
         return i
 
     def fn(self, filename):
-        """Prepend a filename with the project's root directory path.
+        """Prepend a filename with the project's path.
```

Comment:
"The project's path" makes it sound like the path is owned by the project, when actually you could copy paste a project directory and then the new path would be different.

(This may get into something deep about how projects are defined in v2.0 that I may have missed)

```suggestion
        """Prepend a filename with the path to the project.
```

Reply:
How about:
```suggestion
        """Prepend a filename with the project path.
```

Reply:
If you were to copy-paste a project, the copied project's path is different. I don't really understand this distinction.

Reply:
Refactored in de6d897d99ae5865e865b05df04c7f5f16c1b9c7.

Reply:
I agree with the change.